### PR TITLE
Update email styling to work on narrow devices.

### PR DESCRIPTION
**Pull request by lionell-pack-ttd**

### CODE REVIEW

**Comment by lionell-pack-ttd** [comment]
Gmail web client screenshots.
chrome_zxNVa2spGt
chrome_1knsSOT9wE
chrome_rSBzN3wjW0
chrome_lMqZC2hiw6
chrome_KtvqMuSmzv

**Comment by lionell-pack-ttd** [comment]
Gmail mobile with narrow screen (this is a device that had the formatting problem originally reported).
Vysor_gh32eYpe75
Vysor_I6tbT1xKk7
Vysor_UKIQNnr9aK
Vysor_KKqEIuJLin
Vysor_EIYoZZy8Gi

**Review by jingyi-gao-ttd — APPROVED** [review]

File: keycloak/themes/uid2-theme/email/messages/messages_en.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 emailVerificationBody=Please verify your email address\n\nClick the link below to verify this email and complete your account creation for Unified ID.\n\n{0}
 passwordResetSubject=Reset password
-passwordResetBodyHtml=<div style="text-align: center"> <p style="text-align: left"> Hey {0},<br /><br /> Forgot your Password?<br /><br /> We received a request to reset your password for your account.<br /><br /> Please use the button below to reset your password. </p> </div> <div style="margin: 66px 0"> <a href="{1}" style=" border-radius: 5px; font-weight: 700; font-size: 18px; padding: 10px 23px; margin: 0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center; text-decoration: none; " >Reset Password</a > </div>
+passwordResetBodyHtml=<p> Hey {0}, </p> <p> Forgot your Password? </p> <p> We received a request to reset your password for your account. </p> <p> Please use the button below to reset your password. </p> <div class="footer"> <a href="{1}" class="button" >Reset Password</a > </div>
```
Comment (alex-yau-ttd) [inline keycloak/themes/uid2-theme/email/messages/messages_en.properties:5]: I know this copy isn't new but the capitalised "Password" looks odd to me. What about just `Forgot your password?`

Comment (lionell-pack-ttd) [reply keycloak/themes/uid2-theme/email/messages/messages_en.properties]: I checked the designs and looks like Kristen has removed that line altogether - so it's gone now :)

**Review by alex-yau-ttd — APPROVED** [review]

**Pull request merged**